zerver/actions/message_send.py

```diff
@@ -972,6 +975,36 @@ def do_send_messages(
                     )
                     send_request.automatic_new_visibility_policy = new_visibility_policy
 
+            # Set the visibility_policy of the users mentioned in the message
+            # to "FOLLOWED" if "automatically_follow_topics_where_mentioned" is "True".
+            user_mentions = (
+                send_request.rendering_result.mentions_user_ids & send_request.active_user_ids
+            )
```

Comment:
`mentions_user_ids` covers personal and group mention only, right? -- We decided to include topic and stream wildcard mentions too.

Reply:
What is the syntax for topic mentions?

Reply:
I feel like the rule that we want is "Users for whom this message was processed as a mention", including processing of `wildcard_mentions_notify`?

Reply:
So in case of wildcard mentions, user_profiles who have ```wildcard_mentions_notify``` field as True will follow the topic, right?

Reply:
I have made some changes regarding that. Can you please review that?